crates/scouter/src/core/stats.rs

```diff
@@ -0,0 +1,94 @@
+use ndarray::{Array2, ArrayView2};
+use ndarray_stats::CorrelationExt;
+
+pub fn compute_correlation_matrix(drift_data: &ArrayView2<f64>) -> Array2<f64> {
+    let corr = drift_data.t().pearson_correlation().unwrap();
```

Comment:
Is there a reason you chose to use Pearson here?

Reply:
Its what is available in the ndarray_stats library + We really just need to focus on linear relationship between features. It's a quick read for DS to see what features may be correlated with another when drift is detected for a feature